src/status_im2/contexts/share/view.cljs

```diff
@@ -15,15 +15,23 @@
 (defn header
   []
   [:<>
-   [quo/button
-    {:icon                true
-     :type                :blur-bg
-     :size                32
-     :accessibility-label :close-shell-share-tab
-     :override-theme      :dark
-     :style               style/header-button
-     :on-press            #(rf/dispatch [:navigate-back])}
-    :i/close]
+   [rn/view {:style style/header-row}
+    [quo/button
+     {:icon                true
+      :type                :blur-bg
+      :size                32
+      :accessibility-label :close-shell-share-tab
+      :override-theme      :dark
+      :on-press            #(rf/dispatch [:navigate-back])}
+     :i/close]
+    [quo/button
+     {:icon                true
+      :type                :blur-bg
+      :size                32
+      :accessibility-label :shell-scan-button
+      :override-theme      :dark
+      :on-press            #(rf/dispatch [:navigate-back])}
+     :i/scan]]
```

Comment:
Nitpick: The two buttons are identical and can be refactored into a function with three arguments, accessibility label, label, and content.

Reply:
Hi @ibrkhalil : Thanks for the review!
I love to abstract common logic and make re-usable functions, Although in this case it seems to be an overkill to make the suggested change.

Reply:
Understood, Thanks Sid!